eng/tools/summarize-impact/test/cli.test.ts

```diff
@@ -0,0 +1,84 @@
+import { describe, it, expect } from "vitest"; //vi
+
+import path from "path";
+
+import { getChangedFilesStatuses } from "@azure-tools/specs-shared/changed-files";
+import { PRContext, LabelContext } from "../src/types.js";
+import { evaluateImpact } from "../src/runner.js";
+
+// const REPOROOT = path.resolve(__dirname, "..", "..", "..", "..");
+
+describe("Check Changes", () => {
```

Comment:
These will get snipped out in favor of more isolated integration tests that rely on fixtures over anything else.

Reply:
They have been updated to skip properly in CI context. More will be populated as I exercise the capabilities of this package.